adapters/syncer_test.go

```diff
@@ -0,0 +1,35 @@
+package adapters
+
+import (
+	"testing"
+
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"github.com/prebid/prebid-server/usersync"
+)
+
+func TestSyncers(t *testing.T) {
+	syncers := make(map[openrtb_ext.BidderName]usersync.Usersyncer)
+	for _, bidderName := range map[string]openrtb_ext.BidderName{} {
+		if _, ok := syncers[bidderName]; !ok {
+			t.Errorf("no syncer exists for adapter: %s", bidderName)
+		}
+	}
+}
+
+// Bidders may have an ID on the IAB-maintained global vendor list.
+// This makes sure that we don't have conflicting IDs among Bidders in our project,
+// since that's almost certainly a bug.
+func TestVendorIDUniqueness(t *testing.T) {
+	syncers := make(map[openrtb_ext.BidderName]usersync.Usersyncer)
+	idMap := make(map[uint16]openrtb_ext.BidderName, 0)
+	for name, syncer := range syncers {
```

Comment:
Given the circular dependency issue: could you either move this test back into the `usersyncers` package for now, or else move the static map into this package to avoid the circular dependency?

Until the `Register`/`panic` changes are in, this test _does_ have some value... and I have a few good reasons not to go that route, so I'd rather treat this PR as an isolated change set while we discuss.

Reply:
Sure thing!